.github/workflows/lintercheck.yml

```diff
@@ -0,0 +1,79 @@
+name: Linter check
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
      - main
```

Reply:
oh wait this is xla

Reply:
Not yet, I'll do the renaming on Monday (@JackCaoG proposed last Wed, but I've missed the deadline)

Reply:
pytorch/xla still uses master branch, so I guess we should leave it as `master` for now?